ProjectName/.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
MR changes should stick to the requirement of the task. It is good to have husky but I don't think it relates to the task of converting to TS. It would be good to create a separate task for adding husky and add it through MR for that task.